Compilation Error

Dear all,

Kratos is not compiling. The error that I receive is:

/home/dcagri/Kratos210319/kratos/utilities/sensitivity_builder.cpp:157:46: error: no matching function for call to ‘forward(std::initializer_list<{anonymous}::sensitivity_builder_cpp::CalculateNodalSolutionStepSensitivities(const {anonymous}::sensitivity_builder_cpp::SensitivityVariables<TDataType>&, Kratos::ModelPart&, Kratos::AdjointResponseFunction&, double) [with TDataType = double]::__lambda1>&)’
             std::forward<TCallable>(local_fun)(*(rContainer.begin() + i));

Regards

--- a/kratos/utilities/sensitivity_builder.cpp
+++ b/kratos/utilities/sensitivity_builder.cpp
@@ -150,7 +150,7 @@ void ParallelForEach(TContainer& rContainer, TCallable&& Fun)
 {
 #pragma omp parallel
     {
-        auto local_fun{Fun};
+        auto local_fun(Fun);
 #pragma omp for
         for (int i = 0; i < static_cast<int>(rContainer.size()); ++i)
         {

